examples/MidiBle/MidiBle.ino

```diff
@@ -17,8 +17,10 @@ void setup()
 {
   MIDI.begin();
 
+#ifdef LED_BUILTIN
```

Comment:
When testing the code on a LED enabled device (eg Adafruit ESP32 feather), the LED is never turned on when connected.

Reply:
Can you remove the chanegs from the .ino files in the PR?